nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/java/org/apache/nifi/web/filter/LogoutFilter.java

```diff
@@ -50,7 +50,7 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
             final ServletContext apiContext = servletContext.getContext("/nifi-api");
             apiContext.getRequestDispatcher("/access/knox/logout").forward(request, response);
         } else {
-            ((HttpServletResponse) response).sendRedirect("../login");
+            ((HttpServletResponse) response).sendRedirect("../nifi/login");
```

Comment:
Can you test, `sendRedirect("login");`? I think the reason it breaks is because we are already in the `nifi` directory, but the `..` makes us leave it.

Reply:
Sure, I tested it, and it works as well. Thank you for checking it.